src/App.css

```diff
@@ -26,7 +26,7 @@ main {
   top: 50%;
   left: 50%;
   transform: translate(-50%, -50%);
-  box-shadow: 0 0 16px 11px #29986cad;
+  box-shadow: 0 0 16px 11px #1a0f3ead;
 }
```

Comment:
- [ ] You have done a great job, especially on the design part, but some requirements are not adhered to. The boxes on the home page are smaller than the given designs. Kindly try as much as you can to make your project look the same as the given design.

![design](https://github.com/arnoldnekemiah/crypto-market/assets/53124548/4a4d3b22-930e-4b5b-b8ee-5ff9984f1ba1)

![box](https://github.com/arnoldnekemiah/crypto-market/assets/53124548/f93465c4-61c7-4121-9acb-bd18fd01560e)


![yours](https://github.com/arnoldnekemiah/crypto-market/assets/53124548/fd8f5a86-3ee4-498c-8e9e-6a78cdce4713)